pages/home.html

```diff
@@ -62,15 +66,17 @@
     </a>
     Features
   </h2>
-  <ul>
-    <li>Simulates a 1-on-1 conversation with a tutor.</li>
-    <li>Able to give targeted feedback to learners based on their answers.</li>
-    <li>Allows learners to leave feedback on explorations.</li>
-    <li>Explorations are embeddable in any webpage.</li>
-    <li>Explorations can be played on mobile devices (except for some of the more complex interactions).</li>
-    <li>The exploration editor supports full version history, multiple authors, parameters and analytics dashboards.</li>
-    <li>There are extension frameworks for building new interactions and classification rules.</li>
-  </ul>
+  <p>With Oppia, you can:
+    <ul>
+      <li>simulate a 1-on-1 conversation with a tutor,</li>
+      <li>give targeted feedback to learners based on their answers (tutors),</li>
+      <li>leave feedback on explorations (learners),</li>
+      <li>embed explorations in any webpage.</li>
```

Comment:
This part is originally a paragraph:

_With Oppia, you can simulate a 1-on-1 conversation with a tutor, give targeted feedback to learners, ..... ._

But instead of giving readers a big paragraph, we break it into bullet points. Comma at end of each idea means that there are still more ideas below, and the last bulle point is marked by a period (to end the whole paragraph).

This is the rule for citing ideas indicated as an ISO standards. I learnt it while writing a report for school :smile: